### PR TITLE
NOISSUE - Fix coap adapter deployment

### PR DESCRIPTION
**Pull request by blokovi**
Fix coap adapter deployment

Signed-off-by: Ivan Milošević <iva@blokovi.com>

### CODE REVIEW

**Review by drasko — APPROVED** [review]
LGTM

File: charts/mainflux/templates/ingress.yaml [hunk 1]
```diff
@@ -189,3 +185,12 @@ data:
 metadata:
   name: tcp-services
   namespace: ingress-nginx
+---
+apiVersion: v1
+kind: ConfigMap
+data:
+  # CoAP adapter:
```
Comment (drasko) [inline charts/mainflux/templates/ingress.yaml:192]: Do we need this `:` at the end of the comment. Also - is this the best place for this comment, as somehow section for the rpotocol egins abov the line `apiVersion` if I understand well...

**Review by drasko — CHANGES_REQUESTED** [review]

File: charts/mainflux/templates/ingress.yaml (hunk 1 above)
Comment (blokovi) [reply charts/mainflux/templates/ingress.yaml]: I think we don't need this comment at all. Removed.

**Review by drasko — APPROVED** [review]
LGTM

**Review by manuio — APPROVED** [review]
LGTM

**Pull request merged**